schema/viewer.js

```diff
@@ -28,13 +28,17 @@
  * additional fields below.
  */
 
+import CausalityJWT from './causality_jwt';
+import Me from './me';
 import Sales from './sales';
 import { GraphQLObjectType } from 'graphql';
 
 const ViewerType = new GraphQLObjectType({
   name: 'Viewer',
   description: 'A Viewer',
   fields: () => ({
+    causality_jwt: CausalityJWT,
```

Comment:
I thought the `Viewer` was only necessary when trying to access list endpoints with multiple arguments-- seems that `causality_jwt` and `me` will probably just be accessed as single objects, so why do we need them here?